packages/react/src/components/Masthead/MastheadSearch.js

```diff
@@ -575,6 +608,12 @@ MastheadSearch.propTypes = {
    */
   searchOpenOnload: PropTypes.bool,
 
+  /**
+   * `true` to make the search field open in the initial state.
+   * Search will not close on blur, but with close button only.
+   */
+  searchOpenOnloadNoBlur: PropTypes.bool,
```

Comment:
@kennylam strangely I'm not seeing this in the props table in the documentation...

Reply:
@jeffchew I just noticed i forgot to remove it as well, good catch!